packages/components/popover/src/popover.ts

```diff
@@ -43,14 +43,14 @@ export class Popover extends LitElement {
 
   /**
    * The position of popover relative to its anchor.
-   * @type {'top' | 'right' | 'bottom' | 'left' | 'top-start' | 'top-end' | 'right-start' | 'right-end' | 'bottom-start' | 'bottom-end' | 'left-start' | 'left-end'}
+   * @default bottom
    */
   @property() position?: PopoverPosition = 'bottom';
 
   /**
    * When the contents of your popover is too long to be read inline this should be set to true so the user
    * can navigate to the popover content themselves. `aria-details` is always set, regardless of this property.
-   * Read more about this on the accessibility tab.
+   * Read more about this at https://sanomalearning.design/categories/components/popover/accessibility/.
```

Comment:
Maybe adding a link like:
`[accessibility tab](https://sanomalearning.design/categories/components/popover/accessibility/)`
would be better?

Because right now it's not rendering properly on the website:
<img width="910" alt="image" src="https://github.com/user-attachments/assets/04061a73-0a93-4472-a78e-2e1124b136e9" />